Automattic/studio
# Updated Windows app icon

Fixes https://github.com/Automattic/dotcom-forge/issues/6994

## Proposed Changes

- The `studio-app-icon.ico` file now contains different versions at 16, 24, 32, 48, 64, 128, and 256 pixels to avoid issues with Windows auto-scaling the icon.

## Testing Instructions



- Install Studio on Windows
- Check that the icon looks correct in the File Explorer, Settings, and Dock.

## Pre-merge Checklist
